src/lib/output/plugins/AssetsPlugin.ts

```diff
@@ -52,6 +52,7 @@ export class AssetsPlugin extends RendererComponent {
 
         if (this.customCss) {
             if (existsSync(this.customCss)) {
+                this.application.watchFile(this.customCss);
```

Comment:
So in findTsConfigFile, we end up calling watchFile for each checked path... whether or not it was read. Is there a reason to only call watchFile if the path exists here?

Reply:
Yeah, my bad.  I think I was just originally thinking "it's not looking for a file, these are config options that name a specific  file".  D'oh.

Reply:
Fixed in latest push.